src/coreclr/vm/class.cpp

```diff
@@ -2717,20 +2727,25 @@ void EEClass::AddChunkIfItHasNotBeenAdded (MethodDescChunk* pNewChunk)
     if (pNewChunk->GetNextChunk() != NULL)
         return;
 
```

Comment:
This method can stay as is. It is only used to add internal IL stubs that is observable by user code. Touching it has some risk of introducing subtle race conditions.